.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '10'
+- '11'
```

Comment:
I bumped the node version b/c node after I saw the out of memory error in one of the builds. 
I think it should be bumped anyway (I run node v11 on my local but not sure what other use) but will revert this if there are concerns.

Reply:
So I can't find the netlify build where I remember seeing the memory error. I'm happy to revert this for now since its not related to the row control changes in this PR. 
Let me know if thats acceptable.